apkid/rules/elf/protectors.yara

```diff
@@ -162,3 +162,45 @@ rule vkey_elf : protector
   condition:
     is_elf and $libname and 1 of ($vos*) and 1 of ($detection*) and 1 of ($jni*)
 }
+
+rule verimatrix_elf_arm64 : protector
```

Comment:
We're already in the ELF folder so rule names dont need to add the `_elf_` word in their names.

Reply:
Ok, we must update others rules also